tox.ini

```diff
@@ -1,21 +1,21 @@
 [tox]
-envlist = {py27,py35}-django{111}-{sqlite,mysql,postgres},
-          {py35,py36,py37}-django{210,220}-{sqlite,mysql,postgres},lint
-          {py36,py37}-django{300}-{sqlite,mysql,postgres},lint
+envlist = {py36,py37,py38,py39,py310}-django{220}-{sqlite,mysql,postgres},lint
+          {py36,py37,py38,py39,py310}-django{300,310,320}-{sqlite,mysql,postgres},lint
 
 [gh-actions]
 python =
-    2.7: py27
-    3.5: py35
     3.6: py36
```

Comment:
Looking into this more for other projects, I noticed that 3.6 isn't in any list of actively supported python versions: https://devguide.python.org/#status-of-python-branches

This may be something to consider dropping for the purposes of getting the build to pass.

Reply:
This may eliminate unneeded complexity, true. It appeared to be working, with the remaining failures down to changes to django 3.2 requiring secret_key to make connections. I'm happy to remove 3.6 😄

Reply:
Done